pennylane/qinfo/transforms.py

```diff
@@ -385,14 +381,14 @@ def wrapper(*args, **kwargs):
         p = new_qnode(*args, **kwargs)
 
         # In case multiple variables are used, we create a list of cfi matrices
-        if isinstance(j, tuple) and len(j) > 1:
+        if isinstance(j, tuple):
             res = []
             for j_i in j:
-                if interface == "tf":
```

Comment:
I think for those specific tests you were using a list of small tensors instead of one big tensor, e.g. `[tf.Variable(np.pi/8) for _ in range(10)]` instead of `tf.Variable(np.pi/8 * np.ones(10))`. With the latter, the shape is the same as all the other interfaces.

Reply:
great! Thanks for checking that